manifests/globals.pp

```diff
@@ -15,6 +15,7 @@
   Optional[Pattern[/^[57].[0-9]/]] $php_version = undef,
   Optional[Stdlib::Absolutepath] $config_root   = undef,
   Optional[Stdlib::Absolutepath] $fpm_pid_file  = undef,
+  Optional[Stdlib::Absolutepath] $fpm_error_log  = undef,
```

Comment:
`syslog` must be a valid option here

Reply:
syslog is a valid option, see the Archlinux $default_error_log block in the globals.

Reply:
Thank you for your reply about the fpm pools, this has helped.

But i still need to handle the different location of the error log file when using an RHSCL php version.
I know there is already a PR for including RHSCL in php module, but that is taking already a while to get merged.

Will fix the coding style.

Reply:
i am not sure about the `Stdlib::Absolutepath` will work for `syslog` as far as i can remember, the absolute path must start with a `/` see 
- https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/absolutepath.pp
- https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/unixpath.pp
- https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/windowspath.pp

if i set `php::globals::fpm_error_log: syslog` in my `common.yaml` in hiera, i will get an error. so it should be `Optional` or `Optional[Enum['syslog'], Stdlib::Absolutepath]` or something like that (the correct value has to be tested).

Reply:
will take a look at this

Reply:
as you reverted the pool change, please rename this PR `removed default www fpm pool and added parameter for error log file` -> `added parameter for error log file`